internal/auth/auth.go

```diff
@@ -0,0 +1,125 @@
+package auth
+
+import (
+	"html/template"
+	"net/http"
+	"sync"
+	"time"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+var Templates = template.Must(template.ParseGlob("../../internal/public/*.html"))
+
+type User struct {
+	Username string
+	Password []byte
```

Comment:
I believe we decided not to use a password and instead rely on a key stored on the user's thubdrive.